grails-app/controllers/au/org/emii/gogoduck/job/JobController.groovy

```diff
@@ -17,6 +17,7 @@ class JobController {
         }
         else {
             jobExecutorService.register(job)
+            job.statusLink = JobPresenter.getAggrUrl(job, { createLink(it) })
```

Comment:
I think `JobPresenter` is taking care of this "dynamic" stuff already - e.g. so it'd make more sense to put it in there IMO.